src/proteus/dswx_hls.py

```diff
@@ -18,6 +19,8 @@
 
 COMPARE_DSWX_HLS_PRODUCTS_ERROR_TOLERANCE = 1e-6
 
+DEM_MARGIN_IN_PIXELS = 10
```

Comment:
Out of curiosity, was there a reason that 10 was chosen? Could be helpful to document this, for when people ask in the future.